tests/e2e.sh

```diff
@@ -100,6 +100,35 @@ test_cluster_simultaneous_start() {
     stop_test_container test1-orig
 }
 
+test_multiple_clusters_restart() {
+    cluster1='--cluster-port 7946 --wireguard-port 51820 --interface wgoverlay --overlay-net 10.10.0.0/16'
+    cluster2='--cluster-port 7947 --wireguard-port 51821 --interface wgoverlay2 --overlay-net 10.11.0.0/16'
+    setup_wireguard='wireguard-go wgoverlay2 2>/dev/null >/dev/null'
+    join_cluster2="nohup /app/wesher --cluster-key 'ILICZ3yBMCGAWNIq5Pn0bewBVimW3Q2yRVJ/Be+b1Uc=' --join test2-orig $cluster2 2>/dev/null &"
```

Comment:
Don't wanna ask too much of someone contributing their free-time, but can we maybe extend `tests/entrypoint.sh` to get the interface name out of `$@`? :sweat_smile: 
This way we can reuse it and keep the tests a bit cleaner.

Also, I suspect the `nohup` might be what's causing the tests to time-out on travis.

Reply:
I removed the nohup and added an argument parsing routine in entrypoint.sh.